File: demo1s/src/main/scala/org/mosquito/demo1s/service/impl/UserServiceImpl.scala
package org.mosquito.demo1s.service.impl

import org.springframework.stereotype.Service

import org.mosquito.framework.jdbc.service.BaseServiceAdapterImpl
import org.mosquito.demo1s.model.User
import org.mosquito.demo1s.dto.UserDto
import org.mosquito.demo1s.mapper.UserMapper
import org.mosquito.demo1s.implictis.UserImplicits
import org.mosquito.demo1s.service.IUserService

/**
 * 业务类
 *
 * @author jcchen
 * @date 2018-5-28
 * 
 */
@Service("userService")
class UserServiceImpl extends BaseServiceAdapterImpl[UserDto, User, UserMapper] with IUserService with UserImplicits {
	
}

File: demo4/src/main/java/org/mosquito/demo4/service/impl/UserServiceImpl.java
package org.mosquito.demo4.service.impl;

import org.springframework.stereotype.Service;

import org.mosquito.framework.jdbc.service.BaseServiceAdapterImpl;
import org.mosquito.demo4.model.User;
import org.mosquito.demo4.dto.UserDto;
import org.mosquito.demo4.mapper.UserMapper;
import org.mosquito.demo4.service.IUserService;

/**
 * 业务类
 *
 * @author jcchen
 * @Date 2018-6-6
 * 
 */
@Service("userService")
public class UserServiceImpl extends BaseServiceAdapterImpl<UserDto, User, UserMapper> implements IUserService {
	
}
